frontend/package.json

```diff
@@ -20,6 +20,8 @@
     "react-dom": "^17.0.1",
     "react-router-dom": "^5.2.0",
     "react-scripts": "4.0.1",
+    "react-signature-pad-wrapper": "^1.3.0",
+    "signature_pad": "^3.0.0-beta.4",
```

Comment:
Signature_pad is kell? Szerintem azt húzza függőségként a react-signature-pad-wrapper magával.

Reply:
töröltem.